.github/workflows/CI-workflow.yml

```diff
@@ -200,6 +195,24 @@ jobs:
           flags: ml-commons
           token: ${{ secrets.CODECOV_TOKEN }}
 
+  Precommit-codecov:
```

Comment:
Why did we add this?

This is failing all the time: https://github.com/opensearch-project/ml-commons/actions/runs/13911015452/job/38928296296

Please take a look @zane-neo